docs/docsite/rst/community/development_process.rst

```diff
@@ -23,6 +23,62 @@ Ansible accepts code via **pull requests** ("PRs" for short). GitHub provides a
 
 Because Ansible receives many pull requests, we use an automated process to help us through the process of reviewing and merging pull requests. That process is managed by **Ansibullbot**.
 
+Backport Pull Request Process
+-----------------------------
+
+After the pull request submitted to Ansible for the ``devel`` branch is
+accepted and merged, the following instructions will help you create a
+pull request to backport the change to a previous stable branch.
+
+.. note::
+
+    These instructions assume that ``stable-2.5`` is the targeted release
+    branch for the backport.
+
+.. note::
+
+    These instructions assume that ``https://github.com/ansible/ansible.git``
+    is configured as a ``git remote`` named ``upstream``. If you do not use
+    a ``git remote`` named ``upstream`` adjust the instructions accordingly.
+
+#. Prepare your devel, stable, and feature branches:
+
+   ::
+
+       git checkout devel
+       git --fetch upstream
+       git merge upstream/devel
```

Comment:
actually a `pull --rebase` is better to make sure any local changes are 'on top', but this is actually not needed at all since we are updating stable-2.5

Reply:
Yeah, part of this is based around my workflow, and my `devel` is never diverged from `upstream/devel`, so a `git merge upstream/devel` always applies correctly.

Reply:
he, i always diverge devel, then push to diff branch